build-system/pr-check/utils.js

```diff
@@ -115,15 +114,13 @@ function printChangeSummary(fileName) {
 }
 
 /**
- * Starts connection to Sauce Labs after getting account credentials
+ * Starts connection to Sauce Labs using account credentials from env vars.
  * @param {string} functionName
  */
 async function startSauceConnect(functionName) {
-  process.env['SAUCE_USERNAME'] = 'amphtml';
-  const response = await requestPromise(
-    'https://amphtml-sauce-token-dealer.appspot.com/getJwtToken'
-  );
-  process.env['SAUCE_ACCESS_KEY'] = response.trim();
+  if (!process.env.SAUCE_USERNAME || !process.env.SAUCE_ACCESS_KEY) {
```

Comment:
Will we be providing these as encrypted secrets in the Travis config? How do these get set in push builds?

Reply:
Yes, encrypted secrets available to push builds but not to PR builds.

https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml

Reply:
Missed that in the PR description, sorry

Reply:
When I looked I didn't see them defined in `travis.yaml`, must have skipped over it

Reply:
After a second check, I still don't see these defined anywhere in `.travis.yaml`.

Reply:
They're defined in the settings for the `ampproject/amphtml` Travis project. That way, forks will not be able to misuse our tokens.

https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings